src/routes/(app)/strafbakken/+page.server.ts

```diff
@@ -0,0 +1,51 @@
+import type { Actions, PageServerLoad } from "./$types";
+import db from "$lib/server/db";
+import { getUser } from "$lib/server/userCache";
+import { fail } from "@sveltejs/kit";
+
+// Load een overview van alle strafbakken
+export const load = (async () => {
+  return {
+    strafbakken: await db.user.findMany({
+      select: {
+        firstName: true,
+        nickname: true,
+        id: true,
+        _count: {
+          select: {
+            StrafbakReceived: {
+              where: {
+                dateDeleted: null,
+              },
+            },
+          },
+        },
+      },
+    }),
+  };
+}) satisfies PageServerLoad;
+
+export const actions = {
+  // Geef een strafbak
+  default: async ({ request, locals }: { request: Request; locals: any }) => {
+    const data = await request.formData();
+
+    const reason = data.get("reason")?.toString() || undefined;
+    const receiverId = Number(data.get("receiver"));
+
+    // Probeer de user te vinden als dat niet lukt om de een of andere reden, dan is het onsuccesvol
+    const session = await locals.getSession();
```

Comment:
Kan je hier niet locals.user gebruiken?
TS vind het vast niet leuk maar zo ver ik weet moet dat wel kunnen. Deze wordt server-side geset in https://github.com/ODDInvictus/ibs3/blob/54cc3c2efcc3dd27881cff9fa4634caaec14b205/src/hooks.server.ts#L19
en moet hier ook beschikbaar zijn

Reply:
vast wel